zmon_appliance/main.py

```diff
@@ -141,6 +145,27 @@ def ensure_image_versions():
     for artifact, image in sorted(ARTIFACT_IMAGES.items()):
         if image != RUNNING_IMAGES.get(artifact):
             docker_run(artifact, image)
+            updated = True
+
+    return updated
+
+
+def docker_cleanup():
+    logger.info('Cleaning up ...')
+
+    output = subprocess.check_output(['docker', 'ps', '-a', '-q', '-f', 'status=exited', '-f', 'status=created'])
+
+    containers = [c for c in output.decode('utf-8').strip().split('\n') if output]
+    if containers:
+        logger.info('Removing exited {} containers {}'.format(len(containers), ' '.join(containers)))
+        subprocess.call(['docker', 'rm', ' '.join(containers)])
+
+    output = subprocess.check_output(['docker', 'images', '-q'])
+    images = [i for i in output.decode('utf-8').strip().split('\n') if output]
+
+    if images:
```

Comment:
this would delete all images (including running ones which will fail with "conflict")